src/main/java/org/apache/ofbiz/ofbiztutorial/usermgr/events/UserEvents.java

```diff
@@ -0,0 +1,132 @@
+        LocalDispatcher dispatcher = (LocalDispatcher) request.getAttribute("dispatcher");
+        // Extract data from the form
+        HttpSession session = request.getSession();
+        GenericValue userLogin = (GenericValue) session.getAttribute("userLogin");
+        System.out.println("userLogin "+userLogin+"========================");
+        String userLoginId = request.getParameter("userLoginId");
+        String password = request.getParameter("password");
+        String confirmPassword = request.getParameter("confirmPassword");
+        String firstName = request.getParameter("firstName");
+        String lastName = request.getParameter("lastName");
+        String middleName = request.getParameter("middleName");
+//        String gender = request.getParameter("gender");
+<<<<<<< HEAD
+//        String birthDate = request.getParameter("birthDate");
+=======
+        String birthDate = request.getParameter("birthDate");
+>>>>>>> b5bdf4e25b3ebee19ea8ad8ad27302fa32a4f9f6
+        String address1 = request.getParameter("address1");
+        String address2 = request.getParameter("address2");
+        String city = request.getParameter("city");
+        String postalCode = request.getParameter("postalCode");
+        String countryGeoId = request.getParameter("countryGeoId");
+        String countryCode = request.getParameter("countryCode");
+        String areaCode = request.getParameter("areaCode");
+        String contactNumber = request.getParameter("contactNumber");
+        String emailAddress = request.getParameter("emailAddress");
+        // Perform Validations
+        if(UtilValidate.isEmpty(userLoginId)){
+            return ServiceUtil.returnError("User Login Id cannot be empty!!").toString();
+        }
+        if(UtilValidate.isEmpty(password)){
+            return ServiceUtil.returnError("Password cannot be left empty!!").toString();
+        }
+        if(UtilValidate.isEmpty(confirmPassword) && !password.equals(confirmPassword)){
+            return ServiceUtil.returnError("Failed to verify password: Confirm password doesn't match the password!!").toString();
+        }
+        if(UtilValidate.isEmpty(firstName)){
+            return ServiceUtil.returnError("First Name cannot be empty!!").toString();
+        }
+        if(UtilValidate.isEmpty(lastName)){
+            return ServiceUtil.returnError("Last Name cannot be empty!!").toString();
+        }
+//        if(UtilValidate.isEmpty(gender)){
+//            return ServiceUtil.returnError("Gender cannot be empty!!").toString();
+//        }
+<<<<<<< HEAD
+//        if (!UtilValidate.isDate(birthDate)) {
+//            return ServiceUtil.returnError("Incorrect Date Format in Birth Date!!!" + birthDate).toString();
+//        }
+=======
+        if (!UtilValidate.isDate(birthDate)) {
+            return ServiceUtil.returnError("Incorrect Date Format in Birth Date!!!" + birthDate).toString();
```

Comment:
Add all errors in a list and return it only one time with all the issues

Reply:
I'll cover the same in the next commit.